helm/sematic-server/values.yaml

```diff
@@ -70,7 +70,7 @@ deployment:
     timeoutSeconds: 1
 
 image:
-  repository: sematicai/sematic-server
+  repository: sematic/sematic-server
```

Comment:
Not sure why we were still using `sematicai`, but I switched this to `sematic`

Reply:
`sematicai` was a user account I created at the begining. `sematic` is the org we created a few weeks ago. You are right that all images should be moved to the `sematic` org repo. We just have to make sure that they are publicly available.